telegram/_bot.py

```diff
@@ -3558,6 +3558,22 @@ async def get_updates(
             "allowed_updates": allowed_updates,
         }
 
+        # The "or 0" is needed for the case where read_timeout is None.
+        if not isinstance(read_timeout, DefaultValue):
+            arg_read_timeout: float = read_timeout or 0
+        else:
+            try:
+                arg_read_timeout = self._request[0].read_timeout or 0
```

Comment:
so the default behaviour if user leaves everything unchanged is that read_timeout for get_updates is now 5 seconds?